sanitize/exempts.go

```diff
@@ -0,0 +1,31 @@
+package sanitize
+
+// Top-level content fields that should not be sanitized
+var topLevelExemptedFields = map[string]struct{}{
+	"Info":          {},
+	"FormatVersion": {},
+	"Konnect":       {},
+	"Transform":     {},
+	"Workspace":     {},
+}
+
+// Fields that should be skipped for specific entities
+// These fields either require special handling (like Keys)
+// or need not be sanitized like plugin.name
+var entityLevelExemptedFields = map[string]map[string]struct{}{
```

Comment:
Would it be better if we make it clear in code which fields need special handling and which are exempted?

Reply:
In the sanitisation code, I will add the special handling. Separate functions for each entityType.
Those entities still need to be exempted to ensure the sanitisation doesn't overwrite the special handling done.